tests/odbc/test-odbc-postgresql.cpp

```diff
@@ -212,6 +212,11 @@ class test_context : public test_context_base
         return !m_verDriver.is_initialized() || m_verDriver < odbc_version(9, 3, 400);
     }
 
+    bool has_full_uint64_support() const override
```

Comment:
I'm still surprised that PostgreSQL doesn't have it, it's the most standard compliant database, so are we really sure that it's wrong here and not us?

Reply:
TBH I'm confused. IIRC I added these opt-out methods because I had multiple failing tests regarding this during implementation of this PR. As I can't reproduce the errors anymore, I removed them again. That's good news I guess.

The only backends that still have issues with some types are
* Firebird which can't store values bigger INT64_MAX correctly. It instead stores an overflown integer value. This mixes up the ordering of values in the uint64_t vector test.
* SQLite which processes integers as 8-byte signed values. It basically has the same issue as Firebird.
* MSSQL ODBC which seems to be the only one of our tested ODBC backends that needs to use a bigger SQL type in order to store unsigned values (i.e. SQL_BIGINT instead of SQL_INTEGER for uint32_t et cetera).
All of those are either fixed or explicitly tested instead of using an opt-out, though.